packages/block-library/src/quote/edit.js

```diff
@@ -3,6 +3,10 @@
  */
 import { __ } from '@wordpress/i18n';
 import { AlignmentToolbar, BlockControls, RichText } from '@wordpress/block-editor';
+/**
+ * Internal dependencies
+ */
+import { Blockquote } from './blockquote';
```

Comment:
Following @gziolo's [comment on the other PR](https://github.com/WordPress/gutenberg/pull/15769#pullrequestreview-240492296), I'm not 100% sure about this naming.

I don't think this should be a standalone component, but having `Blockquote` inside of the `quote` block is a bit confusing to me. I think I'd suggest something along the lines of "Container" or "Wrapper" but I'd like to hear from @gziolo as well.

Reply:
I shared my thoughts already:

> Ideally, this new component would be renamed to `BlockQuotation` and placed inside `@wordpress/components` package (`packages/components/primitives/block-quotation.js`?) together with the corresponding documentation. See the prior art from @koke for `HorizontalRule` implemented in #14361.
> 
> Related MDN page which I used to reason about the name of the component: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

Reply:
You will have to run into the same issue when porting Pullquote block. It is also something that plugin developers will need as well.

Reply:
What I meant is that, while this gets translated into a `blockquote` HTML element in the web version, I don't see it being a good standalone "quote" component right now, like the HorizontalRule is.

The current web implementation only looks like a quote because it's (IMO, wrongly) [stealing the styles for the quote block](https://github.com/WordPress/gutenberg/pull/15482/files#r287723792), which won't work when we want to do pullquote as well, and the native version is defining a non-configurable style as well.

Also, those styles are only applied to the `blockquote` because of the `wp-block-quote` class, but there's no specific mention of `blockquote` in the CSS. From what I see, all `blockquote` provides on the web is semantic markup, but nothing more over a generic `div`. Which is why, more abstractly, I wonder if it would make sense to make this a generic `View`/`Container` element, where one can pass a `tagName` prop for semantic reasons on the web implementation.

We still have the unresolved issue that we have to use inline styles for native, and CSS classes for the web, but otherwise the code could be pretty similar for both:

```jsx
<View tagName="blockquote" className={ className } style={ style }>
    <RichText
        identifier="value"
        // ...
    />
    { ( ! RichText.isEmpty( citation ) || isSelected ) && (
        <RichText
	    identifier="citation"
            // ...
	/>
    ) }
</View>
```